tests/network/vmi_istio.go

```diff
@@ -183,6 +185,43 @@ var _ = SIGDescribe("[Serial] Istio", func() {
 				}, tests.ContainerCompletionWaitTime, time.Second).Should(Succeed(), fmt.Sprintf("all containers should complete in source virt-launcher pod"))
 			})
 		})
+		Describe("SSH traffic", func() {
+			var bastionVMI *v1.VirtualMachineInstance
+			sshCommand := func(user, ipAddress string) string {
+				return fmt.Sprintf("ssh -y %s@%s\n", user, ipAddress)
+			}
+			checkSSHConnection := func(vmi *v1.VirtualMachineInstance, user, vmiAddress string) error {
+				return console.SafeExpectBatch(vmi, []expect.Batcher{
+					&expect.BSnd{S: "\n"},
+					&expect.BExp{R: console.PromptExpression},
+					&expect.BSnd{S: sshCommand(user, vmiAddress)},
+					&expect.BExp{R: fmt.Sprintf("%s@%s's password: ", user, vmiAddress)},
+				}, 60)
+			}
+
+			BeforeEach(func() {
+				bastionVMI = libvmi.NewCirros(
+					libvmi.WithNetwork(v1.DefaultPodNetwork()),
+					libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding([]v1.Port{}...)),
+					libvmi.WithLabel("version", "v1"),
+					libvmi.WithLabel(vmiAppSelectorKey, vmiServerAppSelectorValue),
+				)
+
+				bastionVMI, err = virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Create(bastionVMI)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(console.LoginToCirros(bastionVMI)).To(Succeed())
```

Comment:
Usually we do - `bastionVMI = tests.WaitUntilVMIReady(serverVMI, console.LoginToCirros)`. Since is doesn't take a lot of time to run a `cirros` vm I believe that skipping the phase waiting for the vm to be started should be ok, but I prefer the code to be consistent.

Reply:
Done